docs/revsets.md

```diff
@@ -157,6 +157,25 @@ For example:
 'user(x)' = 'author(x) | committer(x)'
 ```
 
+### Built-in Aliases
+
+The following aliases are built-in and might be relied on to exist. In every case,
+these functions are defined as aliases to allow you to overwrite them in specific
+situations.
+
+* `trunk()`: Resolves to the head commit for the trunk branch of the `origin`
+  remote. The branches `main`, `master`, `root`, and `trunk` are tried in order.
+  If none of the branches exist, it evaluates to nothing.
```

Comment:
`root` always exists, so you can strike this sentence when you've updated the alias to use that symbol instead